sklearn/feature_extraction/text.py

```diff
@@ -256,10 +256,20 @@ def build_preprocessor(self):
             return strip_accents
 
     def build_tokenizer(self):
-        """Return a function that splits a string into a sequence of tokens"""
+        """Return a function that splits a string into a sequence of tokens
+
+        If there is a capturing group in token_pattern then the
```

Comment:
This should be documented under `token_pattern` in the docstrings of `CountVectorizer`, `TfidfVectorizer` and `HashingVectorizer`